zeebe.js

```diff
@@ -0,0 +1,96 @@
+import axios from "axios";
+import { getAccessToken } from "./auth.js";
+
+const authorizationConfiguration = {
+  clientId: process.env.ZEEBE_CLIENT_ID,
+  clientSecret: process.env.ZEEBE_CLIENT_SECRET,
+  audience: process.env.ZEEBE_AUDIENCE
+};
+
+// An action that assigns a user task.
+async function assignUser([assignee]) {
```

Comment:
```suggestion
async function assignTask([assignee]) {
```

Separate from the other comment, I suggest renaming this function (and the unassign one) to focus on the fact that you're assigning/unassigning a _task_. Task is the subject, the user is an object associated with it.